util/util.go

```diff
@@ -1,10 +1,16 @@
 package util
 
 import (
+	"encoding/json"
 	"fmt"
 	"net/http"
 )
 
+type GeneralResponseObject struct {
```

Comment:
Misschien is het netter om deze gewoon anonymous en inline te gebruiken aangezien hij nu maar op 1 plek wordt gebruikt voor zover ik zie.